CHANGELOG.md

```diff
@@ -230,7 +232,7 @@ There predate the BOLT specifications, and are only of vague historic interest:
 7. [0.5.2] - 2016-11-21: "Bitcoin Savings & Trust Daily Interest II"
 
 [Unreleased]: https://github.com/ElementsProject/lightning/compare/v0.6.2...HEAD
-[0.6.3-rc1]: https://github.com/ElementsProject/lightning/compare/v0.6.3-rc1...HEAD
+[0.6.3]: https://github.com/ElementsProject/lightning/compare/v0.6.3...HEAD
```

Comment:
should this line be:  `https://github.com/ElementsProject/lightning/releases/tag/v0.6.3` ?

Reply:
and `[Unreleased]: https://github.com/ElementsProject/lightning/compare/v0.6.3...HEAD`

Reply:
Yeah, I fuxked up that should not have been here at all.  Will force push